Doc/library/dbm.rst

```diff
@@ -387,7 +389,7 @@ The module defines the following:
       Python's AST compiler.
 
    .. versionchanged:: 3.5
-      :func:`.open` always creates a new database when the flag has the value
+      :func:`open` always creates a new database when the flag has the value
```

Comment:
Does it refer to built-in open() or this module's open()?

Reply:
It refers to `dbm.dumb.open`.